test/graph_test.cpp

```diff
@@ -181,3 +150,96 @@ TEST_P(GraphTest, TestCorrectnessOfReheating) {
     }
   }
 }
+
+TEST_P(GraphTest, MultipleInserters) {
```

Comment:
Does anyone know what this test is? I'm not sure how it got in here

Reply:
you added this with commit "on demand copying of supernodes to disk"

Reply:
Oh! Actually, I do remember what this test is. It's testing the multithreading by setting the number of Graph Workers and the size of each group to higher values. I'll add a comment explaining its purpose